Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -264,5 +264,18 @@ public IActionResult ListUsersOrders(string id)
 
         }
 
+        [Authorize(Roles = RoleCodes.Admin)]
+        public async Task<JsonResult> JsonDetails(int id)
+        {
+            var order = await _dbContext.Orders.SingleOrDefaultAsync(o => o.Id == id);
+
+            if (order == null)
+            {
+                return null;//NotFound(id);

```

Comment:
@jSylvestre why?  I know we still need to setup a not found page but it seems like notfound is the way to go